guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+      caption: For privacy-focused wallets, ask users to confirm trust or offer auto-swap to their default mint when receiving ecash from an unknown mint.
+
+images_federation-guardian-info:
+    - file: federation-guardian-info
+      modalImage: federation-guardian-info@2x
+      alt: Example of a guardian status display in a wallet interface.
+      caption: Example of a guardian status display in a wallet interface.
+
+images_federation-welcome:
+    - file: federation-welcome
+      modalImage: federation-welcome@2x
+      alt: Federation welcome message display in a wallet interface.
+      caption: A welcome message being used durring the new user onboarding process.
+
+images_federation-expiry:
+    - file: federation-guardian-expiry
+      modalImage: federation-guardian-expiry@2x
+      alt: A mobile wallet interface with a screen that shows a countdown timer and a message that the federation will shut down in 15 days.
+      caption: Federation expiration timestamp displayed as a countdown alter when less 30 days remaining.
+    - file: federation-guardian-expiry-join
+      modalImage: federation-guardian-expiry-join@2x
+      alt: A mobile wallet interface with a screen that shows a countdown timer and a message that the federation will shut down in 15 days.
+      caption: Federation expiration timestamp displayed as a data field when more than 30 days remain or the user hasn't yet joined the federation.
+
+images_federation-id:
+    - file: federation-id1
+      modalImage: federation-id1@2x
+      alt: Federation identity and version display in a wallet interface
+      caption: Consider the most important information to display to the user about the federation.
+    - file: federation-id2
+      modalImage: federation-id2@2x
+      alt: Federation descriptions display in a wallet interface
+      caption: Different federation operator may use the description fields differently. Consider this when designing your federation details screen.
+    - file: federation-id3
+      modalImage: federation-id3@2x
+      alt: Federation contact information display in a wallet interface
+      caption: Contact information is optional for the federation operator to include. If there is no contact information, consider not displaying this metadata.
+    - file: federation-id4
+      modalImage: federation-id4@2x
+      alt: Example of important announcements display in a wallet interface
+      caption: A message of the day can consists of anything the federation operator wants to tell the user. It can for example be used to announce a new feature or a upcoming maintenance.
+
+images_cashu-id:
+    - file: cashu-id1
+      modalImage: cashu-id1@2x
+      alt: Mint identity and version display in a wallet interface
+      caption: Consider the most important information to display to the user about the mint.
```

Comment:
What is the "Pubkey" field. Does not seem like something a casual user can make sense of.

Reply:
A version number is also only helpful to highly technical users. Contact info seems more relevant.

Reply:
Pubkey is not yet used, but it’s in the spec so it needs to be removed. Initially when I included the metadata fields for both Fedimint and Cashu I wanted to provide UI examples showing all the metadata fields the protocol supports. Buit as you pointed out, not all metadata fields are equally important. Some are crucial while some are only relevant to technical users. I will revisit this section and mention this distinction. I’ll focus on UI examples that display the most relevant metadata fields for design, and then include a paragraph about the other fields. 
I'll note that while they may be useful for technical users, they are likely irrelevant for most users. It’s best to carefully consider whether or not to include these fields based on your product’s target audience.